.github/workflows/scheduled-upload-crowdin.yml

```diff
@@ -0,0 +1,19 @@
+name: scheduled-upload-crowdin
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: develop
+      - name: Upload Localization to Crowdin by fastlane
```

Comment:
Please update the name of this step since you are not using _fastlane_ but _curl_ directly.